src/main/java/javaserver/Response.java

```diff
@@ -31,9 +31,13 @@ public String getBody() {
 
 	public String formatResponse() {
 		String output = getResponseCode() + System.lineSeparator();
-		output += getHeader() + System.lineSeparator();
+		if (getHeader() != null) {
```

Comment:
You could use the properties (header, statusLine, and body) directly inside this method since it's an instance method of this class. You don't need to use the getters here.